pydy/viz/static/js/dyviz/materials.js

```diff
@@ -14,7 +14,8 @@ DynamicsVisualizer.Materials = Object.extend(DynamicsVisualizer, {
 
         var self = this;
 
-        self.texture_base_path = "static/textures/"
+        // TODO : This `pydy-resources` directory should not be hard coded here!
+        self.texture_base_path = "pydy-resources/textures/"
```

Comment:
We can add a script tag at the end of html files e.g.
```
DynamicsVisualizer.Materials.texture_base_path = {pydy-resources-dir}
```
and can set it like we are doing in display_ipython
```
html = html.format(pydy-resources-dir=self.pydy_reousrces_dir)
```